src/test/kotlin/org/opensearch/replication/integ/rest/UpdateAutoFollowPatternIT.kt

```diff
@@ -294,6 +294,32 @@ class UpdateAutoFollowPatternIT: MultiClusterRestTestCase() {
             .hasMessageContaining(errorMsg)
     }
 
+    fun `test auto follow should fail on indexPattern validation failure`() {
```

Comment:
To ensure we're not regressing, Can you please add the tests for various valid patterns as well?

Reply:
Done